lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -78,14 +79,14 @@
         <hr class="section-divider" />
 
         <div class="start-certificate-generation">
-            <h2>${_("Generate Certificates")}</h2>
+            <h3 class="hd hd-3">${_("Generate Certificates")}</h3>
             <form id="certificates-generating-form" method="post" action="${section_data['urls']['start_certificate_generation']}">
                 % if section_data['html_cert_enabled'] and section_data['active_certificate'] is None:
                     <p>${_("Course certificate generation requires an activated web certificate configuration.")}</p>
                     <input type="button" id="disabled-btn-start-generating-certificates" class="is-disabled" aria-disabled="true" value="${_('Generate Certificates')}"/>
                 % else:
                     <p class="under-heading">
-                        ${_("When you are ready to generate certificates for your course, click Generate Certificates. You do not need to do this<br/>if you have set the certificate mode to on-demand generation.")}
+                        ${Text(_("When you are ready to generate certificates for your course, click Generate Certificates. You do not need to do this if you have set the certificate mode to on-demand generation."))}
```

Comment:
There's no need to use "Text" here because you aren't mixing together Text and HTML.